control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -2588,6 +2585,17 @@ func TestRun_ValidateLoginToken_SecondaryDatacenter(t *testing.T) {
 			GlobalAuthMethod: false,
 			GlobalToken:      false,
 		},
+		{
+			ComponentName: "terminating gateway",
+			TokenFlags: []string{"-terminating-gateway-name=terminating",
+				"-terminating-gateway-name=gateway",
+				"-terminating-gateway-name=another-gateway"},
+			Roles: []string{resourcePrefix + "-terminating-acl-role-dc2",
+				resourcePrefix + "-gateway-acl-role-dc2",
+				resourcePrefix + "-another-gateway-acl-role-dc2"},
+			GlobalAuthMethod: false,
+			GlobalToken:      false,
```

Comment:
for the login test we don't really benefit from providing multiple gateway names since we're stubbing out k8s call, so I think we can test it with just one name/role